database/__init__.py

```diff
@@ -1,2 +1,3 @@
 from database.handler.component_handler import get_component_list, get_component, add_component, update_component, \
     delete_component
+from database.handler.process_handler import get_process_list, get_process, add_process, update_process, delete_process, add_process_reference, update_process_reference,delete_process_reference
```

Comment:
PEP 8: E231 missing whitespace after ','
PEP 8: W292 no newline at end of file
PEP 8: E501 line too long (194 > 120 characters)

```suggestion
from database.handler.process_handler import get_process_list, get_process, add_process, update_process, \
delete_process, add_process_reference, update_process_reference,delete_process_reference
```

(-> ich glaube so irgendwie?)